kuma/core/middleware.py

```diff
@@ -318,13 +318,10 @@ def process_request(self, request):
         return None
 
 
-if settings.DJANGO_1_10:
-    class RedirectsMiddleware(MiddlewareMixin, OriginalRedirectsMiddleware):
-        """
-        Enables the redirect_urls middleware to be used with both MIDDLEWARE
-        and MIDDLEWARE_CLASSES until a newer version of django-redirect-urls
-        is available that provides this "out of the box".
-        """
-        pass
-else:
-    RedirectsMiddleware = OriginalRedirectsMiddleware
+class RedirectsMiddleware(MiddlewareMixin, OriginalRedirectsMiddleware):
+    """
+    Enables the redirect_urls middleware to be used with both MIDDLEWARE
+    and MIDDLEWARE_CLASSES until a newer version of django-redirect-urls
+    is available that provides this "out of the box".
+    """
+    pass
```

Comment:
I think we should also remove the `try-except` wrapper around the import of `MiddlewareMixin` at the top of this file as well as the top of `kuma/wiki/middleware.py`. So in both files replace:
```python
 # TODO: Remove the try-except wrapper after move to Django 1.10+.
 try:
     from django.utils.deprecation import MiddlewareMixin
 except ImportError:
    MiddlewareMixin = object
```
with:
```python
from django.utils.deprecation import MiddlewareMixin
```